spam.py

```diff
@@ -184,12 +185,22 @@ def role_info(self, your_uid):
         elif your_role in EVIL_ALIGNED_ALL:
             info['original_alignment'] = 'evil'
 
+        valid_fakes = self.players
```

Comment:
Can we break prank mode out into a separate method? This method is getting kind of long.

Reply:
It's difficult (but probably doable) to break the entire prank mode into another method here, because of people who see two different groups in the vision matrix (i.e. evil when there's a Lancelot). So I just broke out the random group selection.